src/modules/Dashboard/pages/Applet/AppletMultiInformant.tsx

```diff
@@ -37,7 +45,35 @@ export const AppletMultiInformant = () => {
     <StyledBody>
       {isLoading && <Spinner />}
       {appletData && (
-        <LinkedTabs hiddenHeader={hiddenHeader} tabs={appletTabs} isCentered={false} />
+        <>
+          <StyledFlexTopCenter
+            gap={theme.spacing(1.6)}
+            margin={`${theme.spacing(1.2)} ${theme.spacing(3.4)}`}
```

Comment:
Could you please use sx? In this case sx={{ gap: ..., m: theme.spacing(1.2, 3.4) }}

Reply:
I had the same comment on the other PR but I missed to update it here, thank you!